stores/libsql/src/vector/index.ts

```diff
@@ -44,13 +61,52 @@ export class LibSQLVector extends MastraVector {
       authToken,
       syncInterval,
     });
+    this.maxRetries = maxRetries;
+    this.initialBackoffMs = initialBackoffMs;
 
     if (connectionUrl.includes(`file:`) || connectionUrl.includes(`:memory:`)) {
```

Comment:
same as above with isLocal and move the code to a private fn

Reply:
I think this makes sense to keep in the constructor as it's setting properties, it's not just a local thing